business/istio_validations.go

```diff
@@ -480,6 +473,62 @@ func (in *IstioValidationsService) fetchDetails(rValue *kubernetes.IstioDetails,
 	}
 }
 
+func (in *IstioValidationsService) fetchAllVirtualServices(istioDetails *kubernetes.IstioDetails, namespace string, errChan chan error, wg *sync.WaitGroup) {
+	defer wg.Done()
+	if len(errChan) == 0 {
+		nss, err := in.businessLayer.Namespace.GetNamespaces()
+		if err != nil {
+			errChan <- err
+			return
+		}
+
+		currentVSs, err := in.fetchVirtualServices(namespace)
+		if err != nil {
+			select {
+			case errChan <- err:
+			default:
+			}
+		} else {
+			for _, ns := range nss {
+				vsList, err := in.fetchVirtualServices(ns.Name)
+				if err != nil {
+					select {
+					case errChan <- err:
+					default:
+					}
+				} else {
+					for _, vs := range currentVSs {
+						var namespaceFound = false
```

Comment:
It sounds non-idiomatic go in here. I'd go for `namespaceFound := false`. However, I guess it is matter of style, tbh.

Reply:
yes, python style :)